dclab/definitions/meta_const.py

```diff
@@ -124,6 +124,40 @@
         ["target duration", float, "Target measurement duration [min]"],
         ["target event count", fint, "Target event count for online gating"],
     ],
+    # All qpi-related keywords
+    "qpi": [
+        # experiment-related qpi metadata, see qpretrieve for details
+        ["wavelength", float, "Laser wavelength [nm]"],
+        ["medium index", float, "Refractive index of Medium"],
+        # post-analysis-related qpi metadata
+        ["software version", str, "Software version used"],
+
+        # calculation of pha and amp from hologram
+        # preparation for Fourier transform
+        ["padding", fbool, "If padding was used"],
+        ["subtract_mean", fbool, "Subtract mean of hologram before FFT"],
+        # pipeline_kws
+        ["filter_name", str, "Fourier filter used"],
+        ["filter_size", str, "Fourier filter size [1/pix]"],
+        ["filter_size_interpretation", str, "How to interpret filter size"],
+        ["scale_to_filter", fboolorfloat, "Crop Fourier image before iFFT"],
+        ["sideband_freq", f1dfloattuple,
+         "Frequency coordinates of the sideband [1/pix]"],
+        ["invert_phase", fbool, "Invert the phase data"],
+
+        ["pixel_size_qpi", float,
+         "Image space pixel size after iFFT [um]. "
+         "Depends on `scale_to_filter`. "
+         "If `scale_to_filter` is False, this does not change."
+         "If `scale_to_filter` is not False, this value will differ from "
+         "`qpi_holo` `pixel size`."
+         ],
```

Comment:
I added a lot of info here, which I assume is not exactly what you meant by documentation, but I wanted to use it as a starting point, because finding a good description is difficult!

Reply:
You could add this description as a comment `#` in this file for now. The doc string should be relatively short so it can be viewed properly in UIs such as Shape-Out.